simphony_ui/ui.py

```diff
@@ -186,12 +216,15 @@ def _run_calc_threaded(self):
         """ Function which will run the calculation. This function
         is only run by the secondary thread
         """
-        return run_calc(
-            self.global_settings,
-            self.openfoam_settings,
-            self.liggghts_settings,
-            self.update_progress_bar
-        )
+        try:
+            return run_calc(
+                self.global_settings,
+                self.openfoam_settings,
+                self.liggghts_settings,
+                self.update_progress_bar
+            )
+        except Exception:
```

Comment:
Take care with setting up a logger here: library code should be careful to do ```logger = getLogger(__name__)``` and then do the logger configuration in the app's main method/function (ie. it should never happen as a side-effect of a regular import statement).  Extra care should be taken here because this is code which is both used as a `__main__` but also imported by tests.